packages/react-menu/src/components/MenuItem/useMenuItemStyles.ts

```diff
@@ -65,10 +66,15 @@ const useStyles = makeStyles({
   }),
 });
 
+const useFocusIndicatorStyle = makeFocusIndicatorStyle(theme => ({
+  outline: `solid 1px ${theme.alias.color.neutral.neutralForeground1}`,
```

Comment:
should we export this around in a mixin instead of each individual component so that's it's easy to stay consistent ? 

i.e. `@fluentui/theme-mixins`

Reply:
This is actually the default style rule for `makeFocusIndicatorStyle` I don't know why I repeated it here in this case